documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,4 +2,247 @@
+- `You cannot submit or void declarations for the %{cohort} cohort. The funding contract for this cohort has ended. Get in touch if you need to discuss this with us`
+- `The entered '#/course_identifier' is not recognised for the given participant. Check details and try again.`
+- `NPQ Courses are no longer supported`
+- `Enter a '#/declaration_date'.`
+- `Enter a valid RCF3339 '#/declaration_date'.`
+- `Enter a '#/evidence_held' value for this participant.`
+- `Enter an available '#/evidence_held' type for this participant's event and course.`
+- `A declaration has already been submitted that will be, or has been, paid for this event`
+- `The property '#/declaration_type' does not exist for this schedule.`
+- `You cannot send retained or extended declarations for participants who began their mentor training after June 2025. Resubmit this declaration with either a started or completed declaration.`
+
+### How does participant band distribution work?
+
+Given the data:
+
+- **Bands:**
+  - **Band A**: 0 to 2000 participants
+  - **Band B**: 2000 to 4000 participants
+  - **Band C**: 4000 participants and up to a maximum of 115% of the contract's target volume
+- **Previous participants**: 2100
+- **Current participants**: 2000
+- **Goal**: Distribute these 2000 participants into the bands.
+
+#### Breakdown
+
+##### **Previous Distribution:**
+- The **first 2000** participants recruited in a cohort were in **Band A**.
+- The **next 100** participants (out of 2100) moved into **Band B**.
+
+##### **Current Distribution (2000 participants):**
+- Since all 2000 participants fit within **Band A’s range (0–2000)**,
+  **Band A now has 0 participants**.
+- The remaining participants (from 2000 onwards) move into **Band B and Band C**:
+  - **Band B** starts at 2000. Since we need to place 2000 participants:
+    - 1900 fit into **Band B** (since the previous 100 were already in Band B, we adjust).
+    - The remaining 100 go into **Band C**.
+
+#### **Final Result:**
+| Band   | Participants |
+|--------|-------------|
+| Band A | 0           |
+| Band B | 1900        |
+| Band C | 100         |
+
+This ensures the correct redistribution of participants across the defined bands.
+
+### Output Fee vs. Service Fee
+
+#### What are Output fees?
+
+Declarations are paid each time a declaration is submitted. Output fee’s make up 60% of the price per participant. This would equate to £600 in possible payments a lead provider could claim for subject to a participant training and a provide submitting an evidence of engagement via the API.
+
+#### What are Service fees?
+
+Are paid on a monthly basis over 2 years. The figures are based on the recruitment targets supplied by the lead provider. This would equate to around £400 in payments over the course of 24 months (£16.66) paid to lead provider for each participant trained.
```

Comment:
Not sure I follow this 🤔 are service fees always £16.66 per month? the sentence before suggests it's based on recruitment targets?

Reply:
Move `Example: How a price per participant of £1,000 is distributed between **Service Fees** and **Output Fees**:` to the top of the section, so that makes more sense? £400 of service fees divided by 24 months of the training period, so £16.66 paid per participant per month?